test/utils/utils.go

```diff
@@ -383,7 +379,10 @@ func tagOutputToMap(imagesOutput []string) map[string]string {
 		if len(tmp) < 2 {
 			continue
 		}
-		m[tmp[0]] = tmp[1]
```

Comment:
This was completely broken: for a list containing `foo:a`, `foo:b`, `foo:c`, ... well, you can figure it out.